dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ContentletIndexAPIImpl.java

```diff
@@ -548,40 +544,47 @@ public void putToIndex(final BulkRequestBuilder bulk, ActionListener<BulkRespons
     }
 
     @Override
-    public void putToIndex(final BulkRequestBuilder bulk) {
-        this.putToIndex(bulk, null);
+    public void putToIndex(final BulkRequest bulkRequest) {
+        this.putToIndex(bulkRequest, null);
     }
 
     @Override
-    public BulkRequestBuilder createBulkRequest(final List<Contentlet> contentToIndex) {
+    public BulkRequest createBulkRequest(final List<Contentlet> contentToIndex) {
         final BulkIndexWrapper bulkIndexWrapper = new BulkIndexWrapper(createBulkRequest());
         this.appendBulkRequest(bulkIndexWrapper, contentToIndex);
         return bulkIndexWrapper.getRequestBuilder();
     }
 
     @Override
-    public BulkRequestBuilder createBulkRequest() {
-        return new ESClient().getClient().prepareBulk().setRefreshPolicy(RefreshPolicy.NONE);
+    public BulkRequest createBulkRequest() {
+        final BulkRequest bulkRequest = new BulkRequest();
+        bulkRequest.setRefreshPolicy(RefreshPolicy.NONE);
+        return bulkRequest;
     }
 
-    public BulkProcessor createBulkProcessor(final BulkProcessorListener bulkListener) {
-        final BulkProcessor.Builder builder = BulkProcessor.builder(
-                new ESClient().getClient(), bulkListener);
+    public BulkProcessor createBulkProcessor(final BulkProcessorListener bulkProcessorListener) {
+        BulkProcessor.Builder builder = BulkProcessor.builder(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'builder' could be declared final](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=3712629)